docs/calendar.md

```diff
@@ -46,10 +46,16 @@ The identifier of a custom calendar must consist of one or more components of be
 ### Protocol
 
 It's also possible for a plain object to be a custom calendar, without subclassing.
-The object must implement the `Temporal.Calendar` methods and have an `id` property.
+The object must implement all of the `Temporal.Calendar` methods except for `fields()`.
 It must not have a `calendar` property, so that it can be distinguished in `Temporal.Calendar.from()` from other Temporal objects that have a calendar.
 It is possible to pass such an object into any Temporal API that would normally take a built-in `Temporal.Calendar`.
 
+Any implementation of the protocol with the required methods will return the correct output from any Temporal property or method.
+However, most other code will assume that custom calendars act like built-in `Temporal.Calendar` objects.
+If you want to interoperate with libraries or other code that you didn't write, then you should implement the `id` property and the `fields()` method as well.
```

Comment:
I don't get why `id` and `fields()` are optional.  What happens if Temporal gets a calendar without those fields and some method that needs them is called (like toString for `.id` or getFields() for fields()?)

Reply:
Note that `id` defers to `toString`, not the other way around.

Reply:
If `fields` isn't present on the object, then %Temporal.Calendar.prototype.fields% is called, which just clones the input array. It's optional to override `fields` if you subclass, so it should be optional to provide it on a plain-object calendar. This is consistent with what we do for the TimeZone methods that are optional to override when you subclass TimeZone.